modules/browser/BrowserDetection.js

```diff
@@ -123,6 +124,8 @@ function _detect() {
     const { name, version } = bowser;
 
     if (name in bowserNameToJitsiName) {
+        logger.info(`This appears to be ${name}, ver: ${version}`);
```

Comment:
I actually really appreciate this logging getting back added. I hadn't realized how much I had gotten used to seeing it.

Reply:
Me too :)